Bugzilla/API/V1/Github.pm

```diff
@@ -221,9 +221,11 @@ sub push_comment {
   my $ref        = $payload->{ref};
   my $repository = $payload->{repository}->{full_name};
   my $commits    = $payload->{commits};
+  my ($branch)   = $ref =~ /refs\/heads\/(.*)$/;
 
-  # Return success early if there are no commits or the ref is not a branch
-  if (!@{$commits} || $ref !~ /refs\/heads\//) {
+  # Return success early if there are no commits
+  # or the branch is not one we are interested in.
+  if (!@{$commits} || $branch !~ /^(?:main|master|releases_v\d+)$/) {
```

Comment:
I think we should be checking the value of the `default_branch` key on the repo object instead of hard-coding `master` and `main` here.

Also, the [behaviour requested in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1821521#c2) is for landings to `release_*` to only set flags and **not** close the bug.

Reply:
> I think we should be checking the value of the `default_branch` key on the repo object instead of hard-coding `master` and `main` here.

Cool. Will change to use that.

> Also, the [behaviour requested in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1821521#c2) is for landings to `release_*` to only set flags and **not** close the bug.

Yeah it was brought to my attention yesterday by ryan that needed to be changes so I filed it as a separate bug just now https://bugzilla.mozilla.org/show_bug.cgi?id=1821694